.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,92 @@
+name: 🐞 Bug
+description: Create a report to help us improve
+title: "🐞 Bug"
+labels: ["🐞 bug", "pending triage"]
+body:
+- type: checkboxes
+  attributes:
+    label: Is there an existing issue for this?
+    description: Please search to see if an issue already exists for the bug you encountered
+    options:
+    - label: I have searched the existing issues
+      required: true
+- type: textarea
+  attributes:
+    label: Describe the bug
+    description:  A clear and concise description of what the bug is
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Steps To Reproduce
+    description: Steps to reproduce the behavior
+    placeholder: |
+      1. Go to'...'
+      2. Click on'...'
+      3. Scroll down to'...'
+      4. See error
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Expected Behavior
+    description: A clear and concise description of what you expected to happen
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Screenshots
+    description: |
+      If applicable, add screenshots to help explain your problem
+
+      Tip: You can attach images or log files by clicking this area to highlight it and then dragging files in
+  validations:
+    required: false   
+- type: textarea
+  attributes:
+    label: Is faced issue/bug related to vue simulator?
+    description: A clear and concise description of what you have faced issue with
+  validations:
+    required: true 
+- type: textarea
+  attributes:
+    label: Used vue simulator with or without backend [optional]
```

you can remove `[optional]` its for your refernce
```suggestion
    label: Used vue simulator with or without backend
```